### ISSUE TITLE
Shiba Tsukune + Restored Heirloom

### ISSUE THREAD

**Issue by Bulkhatos** [gryffon/ringteki#2452]
Hello,

Just want to report that Restored Heirloom does not trigger when trying to resolve the Water Ring with Shiba Tsukune.

Cheers!

**Comment by jeremylarner** [comment]
Thanks for the report - I can't replicate this though. Was there anything else on the board which might have stopped it happening?  If you can find a sequence of steps to reproduce this, please let me know.

**Comment by Bulkhatos** [comment]
Thank you for the reply! 

the sequence of steps are:
heirloom was in discard pile >  conflict phase ends > Shiba Tsukune triggers, I resolve void and water ring > void ring resolves normally > water ring only gives option to bow or unbow - had to bring heirloom manually from discard pile.

normally when using water ring to activate azunami or heirloom it gives a prompt before asking to bow/ unbow. in this case, it skips this prompt and went straight to water ring bow/unbow.

hope it can help. cheers!

Cheers!

**Comment by jeremylarner** [comment]
I tried this, but I got an 'Any Interrupts?' prompt, and heirloom was highlighted. Was there a legal water ring target in play?

**Comment by Bulkhatos** [comment]
Since it was Tsukune skill, trigered after conflict phase, any bowed char
was an eligible target. I will keep an eye out if I am doing something
wrong.

cheers

Em Qui, 6 de set de 2018 13:06, jeremylarner <notifications@github.com>
escreveu:

**Comment by jeremylarner** [comment]
So, I worked out what is going wrong here.  The game nodes don't seem to have been properly patched with the last code updates.  So this bug was fixed a couple of weeks ago (and it's fixed in the code I'm running locally), but it's not fixed on the live server.

So I can't replicate it on my local version of the game, but it's definitely there in live.

Sorry for the confusion, and hopefully we'll be able to update the code on the live servers soon
